mysql-test/suite/rocksdb/t/delete.test

```diff
@@ -7,9 +7,10 @@
 # DELETE IGNORE is covered in delete_ignore test
 #
 
---disable_warnings
-DROP TABLE IF EXISTS t1,t2;
---enable_warnings
+# MyRocks does not support gap locks in REPEATABLE-READ mode, test does not
+# require RR ISO to complete, so lets temporarily alter the ISO to RC
+set @orig_tx_iso=@@session.tx_isolation;
```

Comment:
Here and everywhere, no need to save/restore the session variable, when it's restored at the very end of a testcase